spec/features/anonymous_user_happy_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.